src/RedisClusterOptions.php

```diff
@@ -248,6 +250,19 @@ public function getLibOption(int $option, mixed $default = null): mixed
         return $this->libOptions[$option] ?? $default;
     }
 
+    public function getUsername(): ?string
```

Comment:
```suggestion
    /** 
      * @return non-empty-string 
      */
    public function getUsername(): ?string
```

Reply:
will make the user field not nullable and consider an empty string as 'no user provided'